examples/updatemetadata.py

```diff
@@ -22,7 +22,7 @@
 # For those usages not covered by the Apache version 2.0 License please
 # contact with opensource@tid.es
 #
-author = 'jmpr22'
+author = 'chema'
```

Comment:
I prefer the use of native python syntax for these type of declarations:

        __author__